src/main/resources/jenkins/plugins/git/GitUsernamePasswordBinding/help-credentialsId.html

```diff
@@ -0,0 +1,31 @@
+<div>
+    Set the git username / password credential for <strong>HTTP</strong> and <strong>HTTPS</strong> protocols.
+
+<p>
+<strong>Shell example</strong>
+<pre>
+withCredentials([GitUsernamePassword('my-credentials-id')]) {
```

Comment:
Needs the `credentialsId` argument explicitly named
```suggestion
withCredentials([GitUsernamePassword(credentialsId: 'my-credentials-id')]) {
```